### ISSUE TITLE
Duplicating pull request from youtube-dl to yt-dlp

### ISSUE THREAD

**Issue by nomevi** [yt-dlp/yt-dlp#4202]
### Checklist

- [X] I'm asking a question and **not** reporting a bug or requesting a feature
- [X] I've looked through the [README](https://github.com/yt-dlp/yt-dlp#readme)
- [X] I've verified that I'm running yt-dlp version **2022.06.22.1** ([update instructions](https://github.com/yt-dlp/yt-dlp#update)) or later (specify commit)
- [X] I've searched the [bugtracker](https://github.com/yt-dlp/yt-dlp/issues?q=) for similar questions including closed ones. DO NOT post duplicates
- [X] I've read the [guidelines for opening an issue](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#opening-an-issue)

### Question

I just opened a [pull request](https://github.com/ytdl-org/youtube-dl/pull/31057) in the youtube-dl repository. Since I use yt-dlp and not the original, I also want it implemented here. Should I open a pull request here too or wait for you to rebase?

The README has me confused on what to do. First it says

> The main focus of this project is adding new features and patches **while also keeping up to date with the original project**

Then it says

> Based on youtube-dl 2021.12.17 [commit/8a158a9](https://github.com/ytdl-org/youtube-dl/commit/8a158a936c8b002ef536e9e2b778ded02c09c0fa)

**Comment by pukkandan** [comment]
Open a duplicate PR if you want it to be directly merged into yt-dlp. If you only want it to be pulled into yt-dlp after it has been merged in youtube-dl, you don't need to do anything.

Note that the two projects have differing [contribution guidelines](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#developer-instructions). Your PR will be merged here directly only if you follow our guidelines and address our reviews. On the other hand, if it is being pulled from upstream, the code will be treated as "legacy" and merged as-is without concern for code style

**Comment by nomevi** [comment]
> Could you clarify how that is confusing?

If I'm interpreting it correctly, the last merge was half a year ago, which to me doesn't seem like keeping up to date. I'd suggest adding a note about there possibly being a long time between merges with youtube-dl.

**Comment by nomevi** [comment]
> Open a duplicate PR if you want it to be directly merged into yt-dlp. If you only want it to be pulled into yt-dlp after it has been merged in youtube-dl, you don't need to do anything.
> 
> Note that the two projects have differing [contribution guidelines](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#developer-instructions). Your PR will be merged here directly only if you follow our guidelines and address our reviews. On the other hand, if it is being pulled from upstream, the code will be treated as "legacy" and merged as-is without concern for code style

Thanks for the quick reply. I'll open a pull request here then.

Also, was this stated somewhere that I missed? If that isn't the case, I'd suggest that it is added [here](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#developer-instructions).

**Comment by pukkandan** [comment]
> Also, was this stated somewhere that I missed? If that isn't the case, I'd suggest that it is added [here](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md?rgh-link-date=2022-06-27T12%3A57%3A04Z#developer-instructions).

No. For now, I have linked this issue to the FAQ section of https://github.com/yt-dlp/yt-dlp/issues/3766
